yt/yt/library/auth_server/config.cpp

```diff
@@ -160,6 +170,9 @@ void TOAuthServiceConfig::Register(TRegistrar registrar)
         .Optional();
     registrar.Parameter("user_info_error_field", &TThis::UserInfoErrorField)
         .Optional();
+
+    registrar.Parameter("login_transformation", &TThis::LoginTransformation)
+        .Default();
```

Comment:
This change would solve my issue, but I'm not sure that it would be flexible enough in the future.

For example if I want to have transformations like that:
```
user1@acme.com -> user1
user2@othercompany.com -> user2-othercompany
```
I'm not sure how to do it in one regex.

Maybe we can have a list of replacements, so we wouldn't have to make incompatible change in the future since configs are kinda public api?